sentry/src/payout.rs

```diff
@@ -9,9 +9,14 @@ use primitives::{
 use slog::{error, Logger};
 use std::cmp::{max, min};
 
-type Result = std::result::Result<Option<(ValidatorId, BigNum)>, Error>;
-
-pub fn get_payout(logger: &Logger, channel: &Channel, event: &Event, session: &Session) -> Result {
+pub type PayoutResult = std::result::Result<Option<(ValidatorId, BigNum)>, Error>;
```

Comment:
There is actually no need to rename the result. It's a common practice to have a `type Result` which is used for a whole module and it's easy to rename it when importing it `use payout::Result as PayoutResult` or just use the module as prefix `result_var: payout::Result`

You can check out this guideline, which I don't know by heart, but it's very nice reference for e.g. naming functions, methods, etc.
https://rust-lang.github.io/api-guidelines/